Hamzaoutdoors/js_capstone
# Javasript Capstone Project - Web Based API

# In this Project we  created consumed a [TVmaze API](https://www.tvmaze.com/api) 🎥  and an [involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) 👍  where fetched movie/series shows and creating comments and Likes respectively

### Greetings! 👋🏽

> [Visit the Live demo](https://hamzaoutdoors.github.io/js_capstone/)

## How to test this manually
- Open your terminal
- fetch this branch ```git fetch dev```
- Checkout on this branch ```git checkout  dev``
- Run the following commands
- ```npm run build```
- ```npm start```
- Open the local server.
- The application should run as expected.

**Note: ** _we did not implement the reservations feature since we are only two people in the group_.
